docs/mpcnc/belts.md

```diff
@@ -4,17 +4,18 @@
 
 - Move the gantry all the way to the lower left corner.
 - Paying attention to which way the cable tie heads are facing (for that slick clean look).
-- Tie one end of the belt just enough to fit a cable tie through the loop, teeth facing each other and meshed, teeth for the rest of the belt facing down. Pull tight.
-- Loop the next tie through the belt loop, and the corner piece. Leave this one as loose as possible (1″-1.5″ (25-35mm) gap).
-
+- For each belt, the first cable tie will be setup very loose, so that it can be tightened after the belt is routed, and the cable tie on the opposite side is tightened.
+- If the gantry is oriented with the tool mount facing the lower left corner, it will be best to use the left side (when doing the front and rear belts), and the front side (when doing the left and right belts) for the adjustment cable tie.  There is more room on these sides, and the roller bearing cannot reach all the way to these edges.  If the gantry is oriented differently on your setup, you should use the side with the most room.
```

Comment:
This is a good tip. I wonder if it can be a bit shorter and to the point...

```suggestion
- Leave a longer zip tie loop on the end that the tool faces. The gantry won't interfere with the short zip tie, and it has more room on the side with the tool mount.
```
I haven't authored these instructions much, but I know it is a balancing act, because if they get too long, many people won't even read the whole thing.